cmd/server/main.go

```diff
@@ -52,49 +49,40 @@ func init() {
 
 	viper.BindEnv("use_cached_backend")
 
-	options := istiolog.DefaultOptions()
-
-	if viper.IsSet("log_level") {
-		loglevel := viper.GetString("log_level")
-		parsedLogLevel, err := stringToLogLevel(loglevel)
-
-		if err != nil {
-			fmt.Printf("THREESCALE_LOG_LEVEL is not valid, expected: debug,info,warn,error or none. Got: %v\n", loglevel)
-			os.Exit(1)
-		}
-
-		options.SetOutputLevel(istiolog.DefaultScopeName, parsedLogLevel)
-	}
+	configureLogging()
+}
 
-	if viper.IsSet("log_json") {
-		options.JSONEncoding = viper.GetBool("log_json")
-	}
+func configureLogging() {
+	options := log.DefaultOptions()
+	loglevel := viper.GetString("log_level")
+	parsedLogLevel := stringToLogLevel(loglevel)
+	options.SetOutputLevel(log.DefaultScopeName, parsedLogLevel)
+	options.JSONEncoding = viper.GetBool("log_json")
 
-	if !viper.IsSet("log_grpc") || !viper.GetBool("log_grpc") {
+	if !viper.GetBool("log_grpc") {
 		options.LogGrpc = false
-		grpclog.SetLogger(log.New(ioutil.Discard, "", 0))
+		grpclog.SetLoggerV2(
```

Comment:
Can you split the changes about logging into their own commit separate from the shutdown signalling?